lbaas/assets/AnyApp/index.php

```diff
@@ -0,0 +1,49 @@
+<html>
```

Comment:
Now that I'm looking at it, is AnyApp even used anywhere else? Grep only find these occurrences:
```
$ git grep -i AnyApp
lbaas/assets/AnyApp/index.php:echo "<h1> AnyApp </h1>";
lbaas/main.tf:    any_app_php = base64encode(file("${path.module}/assets/AnyApp/index.php"))
sources/AnyApp/index.php:echo "<h1> AnyApp </h1>";
```
so I see no actual references to `sources/AnyApp`...
Otherwise, I would have checked if hard links work properly in git.
And as another possibility the path refering to AnyApp could be made `${path.module}/../sources/AnyApp/index.php`.
But all that may be unneeded if it's really only used once: we can just remove the version from `sources`.

Reply:
That code in a very generic location is intended to be used by as many places as possible. Removing it in favor of a version sigged inside one specific use case does seem even less sensible than duplicating it. Also, it might be used but in a very counterintuitive way, such as downloaded by a curl call in some shell script that was deployed via cloud-init. The ancients from whom we inherited this, had ways that we rather not dream of. I'd really prefer to use the `/../sources/AnyApp` path if it is supported, so that it's easier to see and maintain in future.

Reply:
I think you convinced me :)
How about this MR https://github.com/syseleven/terraform-examples/pull/24 ...